src/pages/industries/software.jsx

```diff
@@ -0,0 +1,219 @@
+import GrayedCanonicalLogo from './images/logos/canonical.svg';
+import GrayedClickHouseLogo from './images/logos/clickhouse.svg';
+import GrayedCosmonicLogo from './images/logos/cosmonic.svg';
+import GrayedDatadogLogo from './images/logos/datadog.svg';
+import GrayedGitlab from './images/logos/gitlab.svg';
+import GrayedPalantirLogo from './images/logos/palantir.svg';
+import GrayedPlanetscaleLogo from './images/logos/planetscale.svg';
+
+const heroContent = {
+  heading: 'SaaS, Software, and  DBaaS',
+  texts: [
+    "In today's fast-evolving software landscape, teams behind SaaS, standalone software, and DBaaS offerings grapple with networking, security, and scalability challenges. Traditional networking solutions often fail to support distributed systems' complex demands spanning multiple clusters and clouds. Furthermore, the rise of microservices intensifies the need for fine-grained security measures and deep visibility into application communications.",
+    'Cilium is purpose-built to secure and accelerate the delivery of cloud native applications. It leverages eBPF, a revolutionary Linux kernel technology, to provide API-aware networking and security with unparalleled granularity without compromising performance. With Cilium, SaaS, software, and DBaaS teams can establish infrastructure capable of scaling their products globally to reach customers wherever they may be. Cilium enables the creation of secure Kubernetes multi-tenant environments, empowering these companies to meet the challenges of modern software deployment head-on.',
+  ],
+  imageSrc: AuditBee,
+  imageAlt: 'ebeedex audit bee',
+};
+
+const datadogTestimonial = {
+  quotedText:
+    'eBPF and Cilium helped us to push the boundaries both within operations and also with product development. To do things safer, faster and more easily than what we could have with traditional techniques. The overlay features in a single product, compatibility with multiple cloud providers, and ability to just run it. These three things are what made Cilium an obvious choice for us',
+  description:
+    'Datadog uses Cilium to secure and monitor the network traffic of their containerized applications running on Kubernetes',
+  withPerson: true,
+  logo: DatadogLogo,
+  name: 'Laurent Bernaille',
+  role: ' Staff Engineer, Datadog',
+  url: 'https://www.cncf.io/case-studies/datadog/',
+  CTAtext: 'READ THE CASE STUDY',
+};
+
+const clickhouseStats = {
+  description:
+    'ClickHouse turned to Cilium as their preferred networking solution to simplify the process of isolating customers from each other',
+  logo: ClickHouseLogo,
+  stats: [
+    {
+      heading: 'MASSIVE SCALE',
+      subHeading:
+        'Secured 10+PiB of streaming data & 30+ trillion inserted records in the first months of deployment',
+    },
+    {
+      heading: 'MULTI-CLOUD',
+      subHeading: '10,000+ pods across multiple regions and cloud providers',
+    },
+    {
+      heading: 'TIME TO VALUE',
```

Comment:
```suggestion
      heading: 'Time to Value',
```

Reply:
@xmulligan This was done intentionally to highlight and draw attention to the heading for the stats.